client/src/components/CollegeShortlistModal.js

```diff
@@ -0,0 +1,188 @@
+                    })
+                    this.props.closeModal();
+                } 
+                else {
+                    this.setState({
+                        submitting: false,
+                    }, () => {
+                        swal({
+                            title: "Done!",
+                            text: "Your college shortlist has been successfully updated!",
+                            icon: "success",
+                        })
+                    })
+                    this.props.addToShortlist(this.state.existingCollegeName);
+                    this.props.closeModal();
+                }
+            } catch (e) {
+                this.setState({
+                    submitting: false
+                }, () => {
+                    console.log("Error: CollegeShortlistModal#submit", e);
+                })
+            }
+        })
+    }
+
+    render() {
+        var shortlist = this.props.addToShortlist;
+        return (
+            <Modal
+                open={this.props.modalOpen}
+            >
+                <Modal.Header>Select your college</Modal.Header>
+                <Modal.Content>
+                {
+                        this.state.country ? 
+                        <>
+                            <Grid centered>
+                                <Grid.Row>
+                                    <Label>
+                                        {this.state.country}
+                                        <Icon
+                                            onClick={this.deleteCountry}
+                                            name='delete'
+                                        />
+                                    </Label>
+                                </Grid.Row>
+                                <Grid.Row>
+                                    <PooledSingleSelectDropdown
+                                        endpoint={`/drop/colleges/${this.state.country}`}
+                                        allowAddition={false}
+                                        dataType={"College"}
+                                        getInput={this.getCollegeInput}
+                                    /> 
+                                </Grid.Row>
+                            </Grid>
+                        </> : 
+                        <Dropdown
+                            placeholder="Search for the country you're in"
+                            style={{ 'margin': '5px', 'width': '80%'}}
+                            fluid
+                            search
+                            selection
+                            options={this.state.countryOptions}
+                            value={this.state.country}
+                            onChange={this.handleCountrySelection}
+                        />
+                    }
+                </Modal.Content>
+                <Modal.Actions>
+                    <Button 
+                        variant="primary"
```

Comment:
I think using `primary` instead of `variant="primary"` might do! (styles currently not applying)